meson.build

```diff
@@ -1679,6 +1679,35 @@ conf.set10('ENABLE_SSH_USERDB_CONFIG', conf.get('ENABLE_USERDB') == 1 and sshdco
 
 conf.set10('SYSTEMD_SLOW_TESTS_DEFAULT', want_slow_tests)
 
+nts = get_option('timesync-nts')
+if nts != 'disabled'
+        if nts == 'openssl'
+                have_gnutls = false
+        else
+                have_gnutls = conf.get('HAVE_GNUTLS') == 1 and libgnutls.version().version_compare('>= 3.6.0')
+        endif
```

Comment:
please no new code with gnutls. just use openssl

Reply:
I guess that also holds for gcrypt? That would simply the build config greatly.

Reply:
Yes just openssl please